src/Symfony/Component/Notifier/Transport/TransportInterface.php

```diff
@@ -24,6 +25,7 @@ interface TransportInterface
 {
     /**
      * @throws TransportExceptionInterface
+     * @throws UnsupportedMessageTypeException
```

Comment:
Wait, same as https://github.com/symfony/symfony/pull/39466#discussion_r540964955, it makes no sense to document logic exceptions.

Reply:
Yes let's remove it